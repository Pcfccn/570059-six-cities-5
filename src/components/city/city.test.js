import React from "react";
import renderer from "react-test-renderer";
import City from "./city";

describe(`Should City render correctly`, () => {
  it(`active city`, () => {
    const tree = renderer
    .create(<City
      thisCity={`Paris`}
      active={true}
      changeCity={() => {}}
    />)
    .toJSON();

    expect(tree).toMatchSnapshot();
  });
  it(`active city`, () => {
    const tree = renderer
    .create(<City
      thisCity={`Paris`}
      active={false}
      changeCity={() => {}}
    />)
    .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
