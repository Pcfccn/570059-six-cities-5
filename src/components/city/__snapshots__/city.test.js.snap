// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should City render correctly active city 1`] = `
<li
  className="locations__item"
>
  <a
    className="locations__item-link tabs__item  tabs__item--active"
    onClick={[Function]}
  >
    <span>
      Paris
    </span>
  </a>
</li>
`;

exports[`Should City render correctly active city 2`] = `
<li
  className="locations__item"
>
  <a
    className="locations__item-link tabs__item "
    onClick={[Function]}
  >
    <span>
      Paris
    </span>
  </a>
</li>
`;
